chainstate/src/detail/tx_verification_strategy/default_strategy.rs

```diff
@@ -111,10 +102,21 @@ impl TransactionVerificationStrategy for DefaultTransactionVerificationStrategy
             })
             .log_err()?;
 
+        // TODO: reconsider the order of connect txs and check block reward.
```

Comment:
Always when writing a TODO, also add why this TODO came to existence. Please add that this was X before, and it was changed to Y because of Z, and so on.